apollo-compiler/src/test/graphql/com/example/arguments_simple/fragment/HeroDetail.kt

```diff
@@ -68,20 +68,6 @@ interface HeroDetail : GraphqlFragment {
   }
 
   companion object {
-    val FRAGMENT_DEFINITION: String = """
-        |fragment HeroDetails on Character {
-        |  __typename
-        |  friendsConnection(first: ${'$'}friendsCount) {
-        |    totalCount
-        |    edges {
-        |      node {
-        |        name @include(if: ${'$'}IncludeName)
-        |      }
-        |    }
-        |  }
-        |}
-        """.trimMargin()
```

Comment:
hm, why it's removed?

Reply:
Might be an oversight. I think I remembered us saying we don' t need it but I might be confusing with something else. I'll add it back

Reply:
It's back in! I'm still curious, what is the use case for `FRAGMENT_DEFINITION` ?